content/modules/fax-sms.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
is there a way to flag an entry as a draft? (since this is a good example, but the module isn't ready yet to be listed here)

Reply:
yes, `draft: true` should do the trick. I'll update